packages/utils/src/common/sharding.ts

```diff
@@ -18,3 +18,42 @@ export function ensurePubsubTopicIsConfigured(
     );
   }
 }
+
+/**
+ * Given a string, will throw an error if it is not formatted as a valid content topic for autosharding based on https://rfc.vac.dev/spec/51/
+ * @param contentTopic String to validate
+ */
+export function validateContentTopic(contentTopic: string): void {
```

Comment:
nitpick: when the method throws an error on failed check, then I believe the `ensure` prefix is more commonly used.

Reply:
renamed to `ensureValidContentTopic`